packages/apputils-extension/src/index.ts

```diff
@@ -271,7 +314,8 @@ const state: JupyterLabPlugin<IStateDB> = {
     const transform = new PromiseDelegate<StateDB.DataTransform>();
     const state = new StateDB({
       namespace: info.namespace,
-      transform: transform.promise
+      transform: transform.promise,
+      windowName: resolver.name
```

Comment:
What is the prefix that is used for a bare `/lab` workspace?

Reply:
The `/lab` prefix is empty string `''`. So if you look in local storage, the `/lab` state keys all begin with `:jupyterlab...`
<img width="742" alt="screen shot 2018-06-07 at 11 13 03" src="https://user-images.githubusercontent.com/159529/41093668-da813888-6a43-11e8-9ea6-47a187cfcc44.png">
And the keys for `/lab/workspaces/foo` begin with `foo:jupyterlab...`